tox.ini

```diff
@@ -49,6 +49,8 @@ deps =
     jwst: jwst[test] @ git+https://github.com/spacetelescope/jwst.git
     romancal: romancal[test] @ git+https://github.com/spacetelescope/romancal.git
     oldestdeps: minimum_dependencies
+    devdeps: astropy>=0.0.dev0
+    devdeps: numpy>=0.0.dev0
```

Comment:
See https://github.com/spacetelescope/romancal/pull/695#discussion_r1189076034 and https://github.com/spacetelescope/romancal/pull/695#discussion_r1189078751